### DISCUSSION TITLE
HTML Button not Appear in the AR view for iOS Devices

### DISCUSSION THREAD

**Discussion by ankursinhrana** [google/model-viewer#2216] [Q&A]
Why does the HTML Custom button not appear in the AR view for iOS Devices?

**Comment by burungiu** [comment]
On which browser did you test it? I just added the snippet below inside the model-viewer and everything works perfectly.
`<button slot="ar-button" className="button_ar">
          View in AR
      </button>`

**Comment by ankursinhrana** [comment]
actually, I'm trying to Develop WEB-AR. 
test link : https://astronaut-d1.glitch.me
It's completely working on Android Device ( only Chrome Browser) but some issues in iOS devices (safari,chrome) like html button (NEXT) not show on iOS Device for AR View.

**Comment by elalish** [comment]
AR on iOS is through the QuickLook native app, so it only gets your `.usdz` or `.reality` file. None of your HTML makes the jump; that's a feature unique to WebXR.
